docs/fx-core/switch-account-or-subscription-help.md

```diff
@@ -0,0 +1,68 @@
+4. Start provision in the selected environment.
+
+## Backup & Recover
+### Backup
+We will keep all backups in the .backup/.fx folder and name those backups with the current date and time in the format of YYYYMMDDHHMMSS (which is the value of "time" mentioned below) when a backup happens. "env" below indicates the environment you select, which could be local or any remote environment.
+* The backup of `state.{env}.json` will be `state.{env}.{time}.json` in the .backup/.fx/states folder which contains generated resources information of the local or remote environment.
+* `azure.parameters.{env}.json` will be copied and saved to `azure.parameters.{env}.{time}.json` in the .backup/.fx/configs folder if your project contains Azure resources and you have selected a remote environment.
+* The backup of `{env}.userdata` which exists when your project requires AAD will be `{env}.{time}.userdata` in the ./backup/.fx/statesfolder which contains secret information.
+
+### Recover
+If you want to switch back to the account or subscription and reuse resources that have been provisioned before:
+* Sign in with the correct accounts and select the correct Azure subscription.
+* Determine the date and time of the backup that you want to recover.
+* Keep a copy of `state.{env}.json`, `azure.parameters.{env}.json` and `{env}.userdata`.
+* Copy the content of `state.{env}.{time}.json` to `state.{env}.json`.    
+Note: if you want to recover for a remote environment and you have added new features, please edit the value of "provisionSucceeded" to "false" to provision resources required for the newly added features.
+* If `{env}.{time}.userdata` exists in the backup folder, replace the content of `{env}.userdata` with the content of `{env}.{time}.userdata`. 
+* If you want to recover for a remote environment and your project previously contains Azure sources, update the value of "resourceBaseName" and "botServiceName"(delete this key if not exists) to the value defined in `azure.parameters.{env}.{time}.json`.
+* Run provision and deploy again.    
+* Delete the backups when you think there is no need to keep them.
+
+## Error
+### Could not be redirected to the teams web page correctly
+If you have previewed (local or remote) your Teams app in one M365 tenant and then switch to another M365 account, you may encounter errors once the browser is launched when previewing in the new M365 tenant. This happens due to the account info saved in the browser storage.
+
+#### Mitigation
+* Launch browser in incognito mode    
+runtimeArgs are the arguments passed to the runtime executable. It is possible to start the browser in private mode. You could edit the launch configuration by adding `"runtimeArgs": ["--inprivate"]` (for Edge) or `"runtimeArgs": ["--incognito"]` (for Chrome).
```

Comment:
Can we give a more complete example of `launch.json`, developers who's not that familiar with VSC probably won't just get it where to add these two arguments.